.github/workflows/checks.yml

```diff
@@ -0,0 +1,16 @@
+name: Merge check
+
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm install
```

Comment:
I would also name steps, but I guess npm <something> is already pretty straightforward.

Reply:
Ah I added job names, but I see you really can name steps. Let me add this quickly.